### ISSUE TITLE
calwebb_tso_spec2.cfg typo scrtype

### ISSUE THREAD

**Issue by jaytmiller** [spacetelescope/jwst#2316]
I think there's a typo in one or more files:

   ./docs/jwst/flatfield/diffs:+      [[scrtype]]
   ./jwst/pipeline/calwebb_tso_spec2.cfg:      [[scrtype]]

Not sure,  I believe scrtype should be srctype based on the step definitions.

**Comment by hbushouse** [comment]
You're absolutely right: `calwebb_tso_spec2.cfg` has a typo, so that needs to be fixed. I also notice that `srctype` doesn't have an entry at all in the `calwebb_spec2.cfg` file, so we should make sure that all steps have an entry (even if it's blank) in both cfgs.
